### PR TITLE
Make including pw in mail to user optional when created by admin

### PR DESCRIPTION
**Pull request by sovainfo**
Pull Request for Issue #10376  .
#### Summary of Changes

When creating user accounts as administrator and sending an email, it is now optional to include the password in the mail.
This is configurable on the plugin. Sending the email was already optional. 
When sending emails, including the password in the email is now optional.
#### Testing Instructions

Apply this patch
As administrator create an user account.
Verify that the password is included in the mail.
Configure plugin User - Joomla to exlude the password.
Create an user account.
Verify that the password in not in the mail.

This closes issue https://github.com/joomla/joomla-cms/issues/10376

### CODE REVIEW

**Comment by wojsmol** [comment]
@sovainfo see https://github.com/sovainfo/joomla-cms/pull/3

File: administrator/language/en-GB/en-GB.plg_user_joomla.ini [hunk 1]
```diff
@@ -8,11 +8,14 @@ PLG_USER_JOOMLA_FIELD_AUTOREGISTER_DESC="Automatically create Registered Users w
 PLG_USER_JOOMLA_FIELD_AUTOREGISTER_LABEL="Auto-create Users"
 PLG_USER_JOOMLA_FIELD_FORCELOGOUT_DESC="Set to No to disable this."
 PLG_USER_JOOMLA_FIELD_FORCELOGOUT_LABEL="Force Logout for all Sessions?"
+PLG_USER_JOOMLA_FIELD_INCLUDEPASSWORD_DESC="Inlude the password when sending a mail to the created user."
```
Comment (andrepereiradasilva) [inline administrator/language/en-GB/en-GB.plg_user_joomla.ini]: "Include password in the email sent to the created user."?

**Comment by sovainfo** [comment]
Thank you, @wojsmol @andrepereiradasilva

**Comment by BurtNL** [comment]
I have tested this item :white_check_mark: successfully on dd4e45760514d8f77517df56c56edc40e2bc34e1<br /><br />Tested successfully. All works as described.
Without the patch: username and password will be send by email.
With the patch applied: send password is optional and will not be send if ticked 'No'.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10462">issues.joomla.org/joomla-cms/10462</a>.</sub>

**Comment by brianteeman** [comment]
I dont understand why you did it this way in the plugin and didnt use the existing option

screen shot 2016-05-17 at 04 11 08<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10462">issues.joomla.org/joomla-cms/10462</a>.</sub>

**Comment by sovainfo** [comment]
Disagree on using the option meant for the registration process. The registration process is different from the account creation process by an administrator. Consider that you should be able to configure that setting for each process. The settings can be the same or different. Setting it for one process has no bearing on the other. That is the functionality I think should be provided.

**Comment by infograf768** [comment]
I agree this should be done in the plugin as it is already there that sending mail is defined.
@sovainfo Can you correct the conflicts?

**Comment by sovainfo** [comment]
Tried to fix the conflicts but apparently failed:
- copy content staging/joomla.xml
- add field 
- edit patch-5/joomla.xml
 Same thing for joomla.php

Maybe somebody else has more luck!

**Comment by wojsmol** [comment]
@sovainfo Please revert to c91e113777574a0fb5e4ff4ad7888cc6ad6b4739.Then rebase will be simple

**Comment by sovainfo** [comment]
@wojsmol Thanks for trying to help. Using GitHub exclusively, haven't found proper way to resolve conflicts, revert or rebase. The only way I know is to close and create a new PR.

Very disappointing if that is what needs to be done. Will wait for feedback before giving up and closing!

**Comment by wojsmol** [comment]
@sovainfo You cen try http://www.hpique.com/2013/09/updating-a-fork-directly-from-github/

**Comment by wojsmol** [comment]
@sovainfo See https://github.com/joomla/joomla-cms/compare/staging...wojsmol:10462-rebase - current staging with thIs PR.

**Comment by sovainfo** [comment]
@wojsmol Thanks, but neither of them help. The procedure is incorrect and can't do anything with the rebase you are pointing to!

Unbelievable for a product that is supposed to make these things easier!

**Comment by roland-d** [comment]
@sovainfo Is this PR to be redone or can you fix the conflicts?

**Comment by sovainfo** [comment]
Can't fix the conflicts!

**Comment by wojsmol** [comment]
@sovainfo Are you planning redo this PR based on the current staging?

**Comment by sovainfo** [comment]
Yes, probably after the release of J360.

**Comment by brianteeman** [comment]
Please see #10990

**Pull request closed without merging**